File: AD Project SA 47 - Team 10 (WebApi)/ADProject_Team10_WebApi/ADProject_Team10_WebApi/Models/ScaleDownModels/StationeryCollectionDetails.cs
/* Author: Nguyen Ngoc Doan Trang */

using System;

namespace ADProject_Team10_WebApi.Models.ScaleDownModels
{
    public class StationeryCollectionDetails
    {
        private String collectionPoint;
        private String status;
        private String date;
        private String stationeryDescription;
        private int quantityNeed;
        private int quantityDisbursed;
        private String itemStatus;
        private String requestDate;

        public string CollectionPoint { get => collectionPoint; set => collectionPoint = value; }
        public string Status { get => status; set => status = value; }
        public string Date { get => date; set => date = value; }
        public string StationeryDescription { get => stationeryDescription; set => stationeryDescription = value; }
        public int QuantityNeed { get => quantityNeed; set => quantityNeed = value; }
        public int QuantityDisbursed { get => quantityDisbursed; set => quantityDisbursed = value; }
        public string ItemStatus { get => itemStatus; set => itemStatus = value; }
        public string RequestDate { get => requestDate; set => requestDate = value; }
    }
}

File: AD_Project_Android_SA47-Team10/app/src/main/java/com/example/kaise/adproject_team10/Entities/StationeryCollectionDetails.java
package com.example.kaise.adproject_team10.Entities;

import java.util.HashMap;
import java.util.List;

/*created by Nguyen Ngoc Doan Trang*/

public class StationeryCollectionDetails {

    private String date;
    private String collectionPoint;
    private String status;
    private List<Item> items;

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getCollectionPoint() {
        return collectionPoint;
    }

    public void setCollectionPoint(String collectionPoint) {
        this.collectionPoint = collectionPoint;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }
}